eox_nelp/edxapp_wrapper/test_backends/event_routing_backends_m_v1.py

```diff
@@ -21,6 +21,7 @@ def get_xapi_constants():
     constants.XAPI_ACTIVITY_QUESTION = "http://adlnet.gov/expapi/activities/question"
     constants.XAPI_VERB_ATTEMPTED = "http://adlnet.gov/expapi/verbs/attempted"
     constants.ATTEMPTED = "attempted"
+    constants.XAPI_ACTIVITY_ATTEMPT = "http://id.tincanapi.com/extension/attempt-id"
```

Comment:
Why did you added it here, and dont use the constant of event routing backends?]
https://github.com/nelc/event-routing-backends/blob/maple-candidate/event_routing_backends/processors/xapi/constants.py#L51

Reply:
Because that package is not part of the requirements, we are using a backend for that library

Reply:
But you could use the mock wrapper for constants??
https://github.com/eduNEXT/eox-nelp/blob/ca068f988334f869928a4e96e2a6d97fd7797802/eox_nelp/edxapp_wrapper/test_backends/event_routing_backends_m_v1.py#L9

https://github.com/eduNEXT/eox-nelp/blob/ca068f988334f869928a4e96e2a6d97fd7797802/eox_nelp/processors/tests/xapi/mixins.py#L181

Reply:
sorry I don't understand your comment

Reply:
Actually more for that my question is for the criteria to use eox-nelp constants or even-routing-backend constants?

Reply:
eox-nelp constants contains values that are different from the event-routing-backends contants or values that are not in upstream

Reply:
So  constant `XAPI_ACTIVITY_ATTEMPT` is not the same as upstream??  https://github.com/nelc/event-routing-backends/blob/maple-candidate/event_routing_backends/processors/xapi/constants.py#L51

Reply:
yes, and I thought it was clear that the code is using the event-routing-backends constants

Reply:
You are right I was confusing the backend with the import of the code. sorry for my confusion and your time clarifying, I thank you.